### DISCUSSION TITLE
Got "Deceptive site ahead" warning from google

### DISCUSSION THREAD

**Discussion by slikas** [authelia/authelia#3363] [Q&A] [answered]
I'd been using my site for a year with basic auth just fine.
After a day implementing authelia I got this warning across browsers and platforms. This seems to be the final clue before the circular "Learn more"

> Security issues
> error
> 1 issue detected
> Google has detected harmful content on some of your site’s pages. We recommend that you remove it as soon as possible. Until then, browsers such as Google Chrome will display a warning when users visit or download certain files from your site.

> Detected Issues
> 
> Deceptive pages
> Description
> These pages attempt to trick users into doing something dangerous, such as installing unwanted software or revealing personal information. [Learn more](https://support.google.com/webmasters/answer/9044101#phising)
> Sample URLs
> N/A

Is it because authelia's login form appears to be "deceptive", "engage in social engineering", or I was hacked? My site is for personal use only, but dynamic ips and vpns may have messed things up

And what to do? Stop using authelia? Buy a new domain? Scanning viruses?

**Comment by james-d-elliott** [comment]
Could you clarify what you mean by basic auth? Also I'm kind of unsure since I've never see this with Authelia. Can you take a screenshot of how this error appears in the browser and show some more context? Can you tell me if this occurs if you install a container like `whoami` and protect it with Authelia? Does it happen on the Authelia page itself?

**Comment by BOZG** [comment]
Not much help as the logs are long gone but I had this warning on one of my domains at some point last year. Google reacted to the redirect to Authelia for whatever reason. I requested a review and explained that it was a redirect to an authentication server. They removed the warning without any questions.

Edit: Should have made it clear that the redirect to Authelia being the problem is speculative rather than confirmed. Like with OP, the warning emerged within a day of implementing Authelia (which replaced Keycloak). As I had made an identical switch on another domain a few months previously without any issues, I just put it down to Google being weird so didn't pay much mind to it.

**Comment by crazytimmy** [comment]
This is super old, but still shows unanswered and not closed.

I just implemented Authelia OIDC on my Portainer service and am getting a "Dangerous" site message from Chrome. I added my domain to Google Search Console and requested a review. Hopefully this is the answer.

Nothing stood out in the logs as this seems to be a browser warning. Here's the warning I was getting:
<img width="1618" height="604" alt="Screenshot" src="https://github.com/user-attachments/assets/90c509c6-53c8-4a53-8973-4d9bf17a7684" />
Checking multiple URLs during the login process on the linked page (https://g.co/safebrowsingstatus) results in "No unsafe content found".

Anyway, at least there some more information for future people who run into this.

**Comment by james-d-elliott** [comment, marked as answer]
Thanks for providing more information. It does confirm however this is not an issue with Authelia or something we can provide support for. Only Google can do that, so the answer realistically is to ask Google to fix this.

This is evident by their own scans at https://g.co/safebrowsingstatus showing no issues with Authelia itself.

I'm locking this discussion at this point.
